tests/components/ct-vue-cli/package.json

```diff
@@ -10,7 +10,7 @@
   },
   "dependencies": {
     "core-js": "^3.8.3",
-    "vue": "^3.2.13",
+    "vue": "3.2.36",
```

Comment:
this looks unrelated? We kept it unpinned to see when the world is broken.

Reply:
Since this PR regenerated package.json, we ran into conflicts right away.